subsys/bluetooth/controller/hci_driver.c

```diff
@@ -288,16 +288,42 @@ static int32_t ble_init(void)
 		return err;
 	}
 
-	ble_controller_resource_cfg_t resource_cfg;
+	ble_controller_cfg_t cfg;
 
-	resource_cfg.buffer_cfg.rx_packet_size = 251;
-	resource_cfg.buffer_cfg.tx_packet_size = 251;
-	resource_cfg.conn_event_cfg.event_length_us = 50000;
-	resource_cfg.role_cfg.master_count = 1;
-	resource_cfg.role_cfg.slave_count = 1;
+	cfg.master_count.count = 1;
 
-	err = ble_controller_resource_cfg_set(
-		BLE_CONTROLLER_DEFAULT_RESOURCE_CFG_TAG, &resource_cfg);
+	err = ble_controller_cfg_set(BLE_CONTROLLER_DEFAULT_RESOURCE_CFG_TAG,
+				     BLE_CONTROLLER_CFG_TYPE_MASTER_COUNT,
+				     &cfg);
+	if (err < 0 || err > sizeof(ble_controller_mempool)) {
+		return err;
+	}
+
+	cfg.slave_count.count = 1;
+
+	err = ble_controller_cfg_set(BLE_CONTROLLER_DEFAULT_RESOURCE_CFG_TAG,
+				     BLE_CONTROLLER_CFG_TYPE_SLAVE_COUNT,
+				     &cfg);
+	if (err < 0 || err > sizeof(ble_controller_mempool)) {
+		return err;
+	}
+
+	cfg.buffer_cfg.rx_packet_size = 251;
+	cfg.buffer_cfg.tx_packet_size = 251;
+	cfg.buffer_cfg.rx_packet_count = 10;
```

Comment:
There are some Zephyr's BLE controller configurations like BT_CTRL_RX_BUFFERS, _TX_BUFFERS, etc. Should we inherit them?

Reply:
let's do that later